Sources/Datadog/RUM/DataModels/RUMDataModelsMapping.swift

```diff
@@ -6,14 +6,59 @@
 
 import Foundation
 
-/* Collection of mappings from SDK models to `RUMDataModel` format. */
+/* Collection of mappings from various types to `RUMDataModel` format. */
+
+extension Int {
+    var toInt64: Int64 {
+        return Int64(exactly: self) ?? .max
+    }
+}
+
+extension UInt64 {
+    var toInt64: Int64 {
+        if self > Int64.max {
+            return .max
+        }
+        return Int64(exactly: self) ?? .max
```

Comment:
`return` line seems to cover all the cases, we can remove if clause above

Reply:
Indeed, done 👌